go/libkb/upak.go

```diff
@@ -18,3 +18,36 @@ func BaseProofSet(u *keybase1.UserPlusAllKeys) *ProofSet {
 	}
 	return NewProofSet(proofs)
 }
+
+// checkKIDPGP checks that the user has the given PGP KID valid *now*. Note that it doesn't
```

Comment:
I found the name `found` throughout this PR a little confusing. Here you say `valid`, I like that better as a name. But not a big deal, either is ok.